packages/foam-vscode/jest.config.js

```diff
@@ -126,13 +126,13 @@ module.exports = {
   // setupFiles: [],
 
   // A list of paths to modules that run some code to configure or set up the testing framework before each test
-  // setupFilesAfterEnv: [],
+  setupFilesAfterEnv: ['jest-extended'],
```

Comment:
This was in every `runCLI` invocation, seems basic enough to be part of the configuration.